infra/aws/terraform/prow-build-cluster/terraform.tfvars

```diff
@@ -14,6 +14,8 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+root_account_arn = "arn:aws:iam::468814281478:root"
```

Comment:
Better to look this up.

Reply:
```suggestion
root_account_arn = "arn:aws:iam::468814281478:user/root"
```

Reply:
It doesn't work with `user/`, but I found a way to look up the account ID, so this comment should be addressed now.